<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleDetailDialog</class>
 <widget class="QWidget" name="RoleDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Role Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex"><number>0</number></property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title"><string>General</string></attribute>
      <layout class="QVBoxLayout" name="generalLayout">
       <item>
        <widget class="QGroupBox" name="roleInfoGroup">
         <property name="title">
          <string>Role Information</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelName">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelDescription">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="descriptionEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="permissionsTab">
      <attribute name="title"><string>Permissions</string></attribute>
      <layout class="QVBoxLayout" name="permissionsLayout">
       <item>
        <widget class="QGroupBox" name="permissionsGroup">
         <property name="title">
          <string>Assigned Permissions</string>
         </property>
         <layout class="QVBoxLayout" name="permissionsGroupLayout">
          <item>
           <widget class="QListWidget" name="permissionsList">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="provenanceTab">
      <attribute name="title"><string>Provenance</string></attribute>
      <layout class="QVBoxLayout" name="provenanceTabLayout">
       <item>
        <widget class="ores::qt::ProvenanceWidget" name="provenanceWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ores::qt::ProvenanceWidget</class>
   <extends>QWidget</extends>
   <header>ores.qt/ProvenanceWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
